### PR TITLE
fix: displayValue not updating when field is out of focus

### PR DESCRIPTION
**Pull request by ravise5**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--aem-boilerplate-forms--adobe-rnd.hlx.live/
- After: https://updateDisplayValue--aem-boilerplate-forms--adobe-rnd.hlx.live/

### CODE REVIEW

File: blocks/form/rules/index.js [hunk 1]
```diff
@@ -86,6 +86,9 @@ async function fieldChanged(payload, form, generateFormRendition) {
         if (['number', 'date', 'text', 'email'].includes(field.type) && (displayFormat || displayValueExpression)) {
           field.setAttribute('edit-value', currentValue);
           field.setAttribute('display-value', displayValue);
+          if (document.activeElement !== field) {
+            field.value = displayValue;
+          }
```
Comment (vdua) [inline blocks/form/rules/index.js:91]: shall we add this check when setting value as well in line number 104 ? It is a separate thing to ensure if we move from change event to input event, things would continue to work

**Review by vdua — APPROVED** [review]
LGTM

**Comment by vdua** [comment]
Forms do not work in the URL shared

File: blocks/form/rules/index.js (hunk 1 above)
Comment (ravise5) [reply blocks/form/rules/index.js:91]: This check is added while setting the value.

**Pull request merged**